src/bika/lims/content/referencesample.py

```diff
@@ -42,10 +43,9 @@
 from zope.interface import implements
 
 schema = BikaSchema.copy() + Schema((
-    ReferenceField('ReferenceDefinition',
+    UIDReferenceField('ReferenceDefinition',
         schemata = 'Description',
         allowed_types = ('ReferenceDefinition',),
-        relationship = 'ReferenceSampleReferenceDefinition',
         referenceClass = HoldingReference,
```

Comment:
I think this can be removed

Reply:
Yep, will do so

Reply:
I've removed all `referenceClass` properties that do no longer make sense, as well as `vocabulary_display_path_bound` and `checkbox_bound` properties